documentation/staging/content/managing-domains/domain-lifecycle/scaling.md

```diff
@@ -12,40 +12,77 @@ This document describes approaches for scaling WebLogic clusters in a Kubernetes
+
+#### On-demand, updating the Cluster or Domain directly
+On-demand scaling (the ability to scale a cluster manually) of WebLogic clusters can be done by directly updating the `.spec.replicas` field of the Cluster or Domain resources.
+##### Updating the Cluster directly
+To scale an individual WebLogic cluster directly, one can simply edit the `replicas` field of its associated Cluster resource. This can be done by using `kubectl`. More specifically, you can modify the Cluster directly by using the `kubectl edit` command.  For example:
+```shell
+$ kubectl edit cluster cluster-1 -n [namespace]
+```
+Here we are editing a Cluster named `cluster-1`. The `kubectl edit` command will open the Cluster definition in an editor and allow you to modify the `replicas` value directly. Once committed, the operator will be notified of the change and will immediately attempt to scale the corresponding cluster by reconciling the number of running pods/Managed Server instances with the `replicas` value specification.
 ```yaml
 spec:
   ...
-  clusters:
-    - clusterName: cluster-1
-      replicas: 1
+  clusterName: cluster-1
+  replicas: 1
   ...
 ```
+##### Updating the Domain directly
+To scale every WebLogic cluster in a domain that does not have a corresponding Cluster resource, or that has a Cluster resource which does not specify a `cluster.spec.replicas` field, modify the `domain.spec.replicas` using the `kubectl edit` command:
 
-Alternatively, you can specify a default `replicas` value for all the clusters.  If you do this, then you don't need to list the cluster in the Domain (unless you want to customize another property of the cluster).
+```shell
+$ kubectl edit domain domain1 -n [namespace]
+```
+Here we are editing a Domain named `domain1`. The `kubectl edit` command will open the Domain definition in an editor and allow you to modify the `replicas` value directly. Once committed, the operator will be notified of the change and will immediately attempt to scale the corresponding cluster(s) by reconciling the number of running pods/Managed Server instances with the `replicas` value specification.
 ```yaml
 spec:
   ...
   replicas: 1
   ...
 ```
-In addition, see the helper scripts in the [Domain lifecycle sample scripts]({{< relref "/managing-domains/domain-lifecycle/startup#domain-lifecycle-sample-scripts" >}}) section.
+
+#### Using Domain Lifecycle Sample Scripts
+Beginning in version 3.1.0, the operator provides sample lifecycle scripts, see the helper scripts in the [Domain lifecycle sample scripts]({{< relref "/managing-domains/domain-lifecycle/startup#domain-lifecycle-sample-scripts" >}}) section, which can be used for scaling WebLogic clusters.
```

Comment:
Minor: "domain-lifecycle/startup#domain-lifecycle-sample-scripts" itself is a small section that in turn simply points to "domain-lifecycle/scripts.md". So might as well make the current relref "/managing-domains/domain-lifecycle/scripts.md" instead...